freshmaker/views.py

```diff
@@ -590,7 +590,7 @@ def post(self):
             Content-Type: application/json
 
             {
-                "dist_git_branch": "master",
+                "dist_git_branch": "main",
```

Comment:
I could be nice to change this one. Since this one is an example, I thought that it changing it could perhaps help people to adopt this convention (or at least spot when a project doesn't follow it). You know, giving a "good example".

I think I'll leave it like this, then.